.github/workflows/pr_notifier.yml

```diff
@@ -12,7 +13,7 @@ jobs:
     steps:
     - uses: actions/checkout@a12a3943b4bdde767164f792f33f40b04645d846
     - name: Set up Python 3.8
-      uses: actions/setup-python@v3
+      uses: actions/setup-python@0ebf233433c08fb9061af664d501c3f3ff0e9e20 
```

Comment:
i think this will fail formatting due to trailing ws

Reply:
copying out ci advice re editor/formatting

```console
 Please fix your editor to ensure:

      - no trailing whitespace
      - no preceding mixed tabs/spaces
      - all files end with a newline

```

Reply:
Took care of it. Thanks